tsconfig.json

```diff
@@ -8,9 +8,11 @@
         "resolveJsonModule": true,
         "skipLibCheck": true,
         "declaration": true,
-        "allowSyntheticDefaultImports": true,
         "sourceMap": true,
         "strict": true,
-        "outDir": "build/"
+        "outDir": "build/",
+        "esModuleInterop": true,
```

Comment:
I think we shouldn't need this if we're outputting esm?

Reply:
we do need it for interoping with dependencies that use CJS, like `preferred-pm`